dev/src/components/NavBar.tsx

```diff
@@ -1,49 +1,56 @@
-import React from "react"
-import { useAppState } from "../overmind"
-import { Link } from "react-router-dom"
-import NavBarFooter from "./navbar/NavBarFooter"
-import NavBarCourse from "./navbar/NavBarCourse"
-import { isEnrolled, isVisible } from "../Helpers"
-
+import React from "react";
+import { useActions, useAppState } from "../overmind";
+import { Link } from 'react-router-dom'
```

Comment:
Also inconsistent use of `;` and two different quote types `"` and `'`. As mentioned in PR #613, you can install the ESLint plugin to automagically fix/format the code to remove the `;` at least.